packages/flutter/lib/src/widgets/restoration_properties.dart

```diff
@@ -226,6 +231,10 @@ class RestorableDouble extends RestorableNum<double> {
   /// Creates a [RestorableDouble].
   ///
   /// {@macro flutter.widgets.RestorableNum.constructor}
+  ///
+  /// See also:
+  ///
+  ///  * [RestorableDoubleN] for the nullable version of this class.
```

Comment:
Why did you put all of these "See also" clauses in the constructor docs instead of in the main docs above?  Just curious, I'm not saying you should change it.

Reply:
Actually, there isn't really a good reason. I think I might've accidentally done this instead of putting it in the main docs. I'll go ahead and change it. Thanks for catching this!